components/status_dropdown/status_dropdown.jsx

```diff
@@ -105,6 +120,87 @@ export default class StatusDropdown extends React.PureComponent {
             </FormattedMessage>
         );
     }
+    handleClearStatus = (e) => {
+        e.stopPropagation();
+        e.preventDefault();
+        this.props.actions.unsetCustomStatus();
+    };
+
+    onToggle = (open) => this.props.actions.setStatusDropdown(open);
+
+    renderCustomStatus = () => {
+        if (!this.props.isCustomStatusEnabled) {
+            return null;
+        }
+        const customStatus = this.props.customStatus;
+        const isStatusSet = customStatus && (customStatus.text || customStatus.emoji);
+        const customStatusText = isStatusSet ? customStatus.text : localizeMessage('status_dropdown.set_custom', 'Set a Custom Status');
+        const customStatusEmoji = isStatusSet ?
+            (
+                <span className='d-flex'>
+                    <CustomStatusEmoji
+                        showTooltip={false}
+                        emojiStyle={{marginLeft: 0}}
+                    />
+                </span>
+            ) : (
+                <EmojiIcon className={'custom-status-emoji'}/>
+            );
+
+        const clearButton = isStatusSet &&
+            (
+                <span
+                    className='custom_status__clear'
+                >
+                    <OverlayTrigger
+                        delayShow={Constants.OVERLAY_TIME_DELAY}
+                        placement='top'
+                        overlay={
+                            <Tooltip id='clear-custom-status'>
+                                <FormattedMessage
+                                    id='status_dropdown.custom_status.tooltip_clear'
+                                    defaultMessage='Clear'
+                                />
+                            </Tooltip>
+                        }
+                    >
+                        <button
+                            className='style--none input-clear-x'
+                            onClick={this.handleClearStatus}
+                        >
+                            <i className='icon icon-close-circle'/>
+                        </button>
+                    </OverlayTrigger>
+                </span>
+            );
+
+        const pulsatingDot = !isStatusSet && this.props.showCustomStatusPulsatingDot && (
+            <span className='pulsating_dot'/>
+        );
+
+        return (
+            <Menu.Group>
+                <Menu.ItemToggleModalRedux
+                    text={''}
```

Comment:
I see why we're passing in no text here and then adding the icon and text as children, but this is going to prevent the underlying element from having its `accessibilityLabel` set. That said, I'm honestly not sure if that's a problem since I would've assumed any sort of accessibility software would just read the contents of the menu item. 

I'll see if I can find any more information about this, but there's probably nothing to change here since it might just be a side effect of whoever added that not having a full understanding of how aria labels should be used. I'll let you know if something does actually need to change here.